roles/lib_openshift/src/doc/sdnvalidator

```diff
@@ -0,0 +1,27 @@
+# flake8: noqa
+# pylint: skip-file
```

Comment:
@rhcarvalho, The reason we skip those files is that they are fragments and do not pass linters on their own.  The doc files are just strings.  They could be .py but since they are fragments and only contain doc, I left them without an extension.

> Sad to see that lib_openshift simply bypasses both flake8 and pylint...

I'm not sure I understand this.  `lib_openshift/library` is put through flake8 and pylint as well as `lib_openshift/src/test`.  Its only the fragments that are not.